communication.rst

```diff
@@ -99,21 +99,99 @@ We have our own `Discourse`_ forum for both developers and users. This forum
+
+Tutorials for new users
+'''''''''''''''''''''''
+To start a topic or participate in any discussions in the forum, sign up and
+create an account using an email address or GitHub account. You can do so by
+clicking the "Sign Up" button on the top right hand corner of the `Discourse`_
+main page.
+
+The Python Discourse `Quick Start <https://discuss.python.org/t/python-discourse-quick-start/116>`_
+compiled by `Carol Willing <https://discuss.python.org/u/willingc/>`_ gives you
+a quick overview on how to kick off Python Discourse.
+
+We recommend new users getting familiarised with the forum by going through Discobot tutorials.
+These tutorials can be activated by replying to a welcome message from "discourse
+Greetings!" received under Notifications and Messages in your user account.
+
+* Click on your personal account found on the top right hand corner of the page.
+* The dropdown menu will show four different icons: 🔔 (Notifications),
+  🔖 (Bookmarks), ✉️ (Messages), and 👤 (Preferences).
+* Select either Notifications or Messages.
+* Open the "Greetings!" message sent by Discobot to start the tutorial.
+* Ensure that you read through the `Python Code of Conduct <https://discuss.python.org/faq>`_.
+  We are to be open, considerate and respectful to all users in the community.
+
+
+
+Reading topics
+'''''''''''''''
+Click a topic title and read down the list of replies in chronological order,
+following links or previewing replies and quotes as you go. Use your mouse to
+scroll the screen, or use the timeline scroll bar on the right which also shows
+you how far through the conversation you’ve read. On smaller screens, select the
+bottom progress bar to expand it.
+
+
+Notifications
+'''''''''''''
+
+Following categories (Category notifications)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Notifications can be set for individual categories and topics. To change any of these
+defaults, you can either go to your user preferences, or visit the category
+page, and use the notification button 🔔 above the topic list,
+on the top right hand corner of the category page beside the “+ New Topic” button.
+
+Clicking on the Notification control 🔔 will show a drop-down panel with 5
+different options: Watching, Tracking, Watching First Post, Normal, and Muted.
+All categories are set by default in Normal mode where you will only be notified
+if someone mentions your @name or replies to you.
+
+Following individual threads (Topic notifications)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To follow any individual topics or threads, you can adjust your notifications
+through the notification button 🔔 found on the right of the topic at the end
+of the timeline. You can also do so at the bottom of each topic.
+Select “Watching” and you will be notified when there is any new updated reply
+from that particular thread.
+
+Customising notifications on user preference
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+<<<<<<< HEAD
```

Comment:
There is a merge conflict here.

Reply:
Oops! Clicked too fast. Conflict resolved.